docs/qubit-routing/qaoa-qubit-routing.md

```diff
@@ -0,0 +1,125 @@
+# Qubit Routing for QAOA
+
+OpenQAOA supports custom qubit routing solutions specific to QAOA circuits as part of the QAOA workflow. 
+
+## What is Qubit Routing?
+Qubit Routing is a key step in circuit compilation that maps a Quantum Circuit defined over logical qubits to the physical qubits in a Quantum Computer. Most of currently available Quantum Computers have sparsely connected physical qubits. Therefore, in order to implement an entaglement operation between physical qubits $i, j$ that are not connected on the QPU, we make use of $SWAP$ gates. A SWAP gate is an entangling operation between two qubits $a, b$ that swaps the quantum state of the two qubits. Mathematically, it can be described as follows:
```

Comment:
Is the SWAP gate an entangling operation?

Reply:
good question haha! Let me think about 😅